backend/bluf-module/config/langchain_ai.py

```
@@ -1,58 +1,95 @@
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../..")))
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-from lib.constants import LANGCHAIN_API_KEY, OPENROUTER_API_KEY, OPENAI_API_KEY
+from langchain_core.language_models import BaseChatModel
+from langchain_core.messages import HumanMessage, SystemMessage
 from config.rich_logging import logger as log
+from config.llm_config import (
+    Providers,
+    PROVIDER_CONFIG,
+    ProviderConfig,
+    Models
```

Ruff (0.8.2)

`config.llm_config.Models` imported but unused

Remove unused import: `config.llm_config.Models`

(F401)